lib/usdUfe/ufe/UsdContextOps.cpp

```diff
@@ -417,6 +465,61 @@ Ufe::ContextOps::Items UsdContextOps::getItems(const Ufe::ContextOps::ItemPath&
     return items;
 }
 
+// Adds the special Bulk Edit header as the first item.
+void UsdContextOps::addBulkEditHeader(Ufe::ContextOps::Items& items) const
```

Comment:
Helper function so that it can be called from LookdevX plugin which also overrides contextOps and sometimes uses only their context menu (not calling ours).